### PR TITLE
clean up setup.py as best as possible

### PR DESCRIPTION
**Pull request by mattmilten**
using black formatting and some more pythonic string and path manipulations

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -2,128 +2,128 @@
...
+        r'^__version__\s*=\s*[\'"]([^\'"]*)[\'"]', initfile.read(), re.MULTILINE
+    ).group(1)
 
 try:
     from Cython.Build import cythonize
 except ImportError as err:
     # if cython is not found _and_ src/pyscipopt/scip.c does not exist then we cannot do anything.
-    if not os.path.exists(os.path.join(packagedir, 'scip.c')):
-        print('Cython is required')
-        quit(1)
```
Comment (fschloesser) [inline setup.py:83]: did you take this line out on purpose?

**Review by fschloesser — COMMENTED** [review]
apart from the small question this looks good to me

File: setup.py (hunk 1 above)
Comment (mattmilten) [reply setup.py:83]: yeah, `sys.exit` is preferred over `quit` according to "the internet" and it has the added benefit of printing a message. So we save one line and have slightly cleaner code.

Comment (fschloesser) [reply setup.py:83]: okay, i cannot find the sys.exit though

Comment (mattmilten) [reply setup.py:83]: image

I just replaced single quotes with double quotes and added sys.exit.

Comment (fschloesser) [reply setup.py:83]: ah yes, i was too close to the blackboard, all is good!

**Pull request merged**